diesel/src/expression/operators.rs

```diff
@@ -63,7 +63,7 @@ macro_rules! __diesel_operator_body {
         expression_ty_params = ($($expression_ty_params:ident,)*),
         expression_bounds = ($($expression_bounds:tt)*),
     ) => {
-        #[derive(Debug, Clone, Copy, QueryId, DieselNumericOps, NonAggregate)]
+        #[derive(Debug, Clone, Copy, $crate::query_builder::QueryId, DieselNumericOps, NonAggregate)]
```

Comment:
It seems as though because `diesel/src/expression/mod.rs` contains `pub trait NonAggregate {}` and `pub use diesel_derives::NonAggregate;` this never came up as an issue at compile time.
Thoughts?

Reply:
It may cause issues in third party code, because they won't necessarily import `diesel::expressions::NonAggregate`

```suggestion
        #[derive(Debug, Clone, Copy, $crate::query_builder::QueryId, diesel::sql_types::DieselNumericOps, $crate::expressions::NonAggregate)]
```